frontend/src/metabase/plugins/index.ts

```diff
@@ -139,6 +139,20 @@ export const PLUGIN_SELECTORS = {
       isDefault: true,
     };
   },
+  getLandingPageIllustration: (_state: State): IllustrationValue => {
+    return {
+      src: "app/img/bridge.svg",
+      isDefault: true,
+    };
+  },
+  getNoDataIllustration: (_state: State): string => {
+    // Can't import any SVG file because it would fail to build static viz
+    return "app/img/no_results.svg";
```

Comment:
It's unfortunate that I need to duplicate `no_results.svg` to `resources/frontend_client/app/img/no_results.svg` because we couldn't build the static viz if we import any SVG file here.

[Example failed tests](https://github.com/metabase/metabase/actions/runs/8437894103/job/23108737428#step:3:324)